src/page_body/structure/blocks/tokens/previews/page_block_token_preview_large.pr

```diff
@@ -111,6 +108,80 @@
         </div>
         {[ inject "page_block_token_themes_badge" context themes /]}
     </div>
-{[ elseif token.tokenType.equals("GenericToken")]}
-    <div class="preview small-text bordered">{{ token.value.text }}</div>
+{[ elseif token.tokenType.equals("FontSize")]}
+    <div class="preview bordered">
+        {[ inject "page_block_token_reference_badge" context token /]}
+        <div class="value-preview" style="font-size: {{ normalizeFontSizeCSS(token.value) }}">Ag</div>
+        <div class="value-footer">
+            {[ inject "page_block_token_value" context token /]}
+        </div>
+        {[ inject "page_block_token_themes_badge" context themes /]}
+    </div>
+{[ elseif token.tokenType.equals("LetterSpacing")]}
+    <div class="preview bordered">
+        {[ inject "page_block_token_reference_badge" context token /]}
+        <div class="value-preview" style="letter-spacing: {{ measureValueToReadableUnit(token.value) }}">Hello</div>
```

Comment:
Just checking out if "Hello" is intended

Reply:
It is, we use the same for letter spacing in the app, and two characters that were there before (Ag) were not enough to show the preview nicely